resources/sdk/purecloudjava/templates/Configuration.mustache

```diff
@@ -20,5 +36,245 @@ public class Configuration {
+    }
+  }
+  
+  private class ConfigurationParser {
+    private String filePath;
+    private FileFormat fileFormat;
+    private Ini iniData;
+    private Map<String, Object> jsonData;
+
+    public ConfigurationParser(String filePath) {
+      this.filePath = filePath;
+      this.fileFormat = FileFormat.Invalid;
+    }
+
+    public boolean read() {
+      try {
+        File configFile = new File(filePath);
+        Ini ini = new Ini(new FileReader(configFile));
+        this.iniData = ini;
+        this.fileFormat = FileFormat.INI;
+      } catch (FileNotFoundException e) {
+        return false;
+      } catch (Exception e) {
+        try {
+          ObjectMapper objectMapper = new ObjectMapper();
+          File configFile = new File(filePath);
+          Map<String, Object> jsonMap = objectMapper.readValue(configFile, Map.class);
+          this.jsonData = jsonMap;
+          this.fileFormat = FileFormat.JSON;
+        } catch (Exception ex) {
+          return false;
+        }
+      }
+      return true;
+    }
+
+    public String getString(String section, String key) {
+      switch (fileFormat) {
+        case INI:
+          return getIniString(section, key);
+        case JSON:
+          return getJsonString(section, key);
+        default: 
+          return "";
+      }
+    }
+
+    public boolean getBool(String section, String key) {
+      switch (fileFormat) {
+        case INI:
+          return getIniBool(section, key);
+        case JSON:
+          return getJsonBool(section, key);
+        default: 
+          return false;
+      }
+    }
+
+    public int getInt(String section, String key) {
+      switch (fileFormat) {
+        case INI:
+          return getIniInt(section, key);
+        case JSON:
+          return getJsonInt(section, key);
+        default: 
+          return 0;
+      }
+    }
+
+    public String getJsonString(String section, String key) {
+      try {
+        Map<String, Object> sectionData = (Map<String, Object>)jsonData.get(section);
+        return sectionData.get(key).toString().trim();
+      } catch (Exception e) {
+        return "";
+      }
+    }
+
+    public boolean getJsonBool(String section, String key) {
+      try {
+        Map<String, Object> sectionData = (Map<String, Object>)jsonData.get(section);
+        return Boolean.parseBoolean(sectionData.get(key).toString().trim());
+      } catch (Exception e) {
+        return false;
+      }
+    }
+
+    public int getJsonInt(String section, String key) {
+      try {
+        Map<String, Object> sectionData = (Map<String, Object>)jsonData.get(section);
+        return Integer.parseInt(sectionData.get(key).toString().trim());
+      } catch (Exception e) {
+        return -1;
+      }
+    }
+
+    public String getIniString(String section, String key) {
+      try {
+        Ini.Section sectionData = iniData.get(section);
+        return sectionData.get(key).trim();
+      } catch (Exception e) {
+        return "";
```

Comment:
I tried using an INI file with the same contents as the README and it wasn't applying the config. I used a debugger on this method when it was called with `getIniString("logging", "log_level")` and there was a NullPointerException being thrown